client/extensions/woocommerce/lib/get-payment-method-details/detailsMappings.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
The filename should probably be `details-mappings.json` instead of camel case for consistency.